src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -387,27 +389,20 @@ private void testPluginAgainst(
         // potential javadoc execution to avoid general test failure.
         if (!ranCompile) {
             runner.run(
-                    mconfig,
+                    Map.of("maven.javadoc.skip", "true"),
                     pluginCheckoutDir,
                     buildLogFile,
                     "clean",
-                    "process-test-classes",
-                    "-Dmaven.javadoc.skip");
+                    "process-test-classes");
         }
         ranCompile = true;
 
         List<String> args = new ArrayList<>();
-        Map<String, String> userProperties = mconfig.userProperties;
-        args.add(
-                String.format(
-                        "--define=forkCount=%s", userProperties.getOrDefault("forkCount", "1")));
         args.add("hpi:resolve-test-dependencies");
         args.add("hpi:test-hpl");
         args.add("surefire:test");
 
         // Run preexecution hooks
-        List<String> testTypes = new LinkedList<>();
-        testTypes.add("surefire"); // default
```

Comment:
This "test types" thing was completely unused, most likely a remnant from the old XML report functionality that we recently ripped out.